packages/zero-client/src/client/zero.ts

```diff
@@ -1084,6 +1115,8 @@ export class Zero<
     this.#socket = ws;
     this.#socketResolver.resolve(ws);
 
+    getBrowserGlobal('window')?.addEventListener('pagehide', this.#onPageHide);
```

Comment:
@arv this breaks react-native, I'm getting error `getBrowserGlobal(\"window\")?.addEventListener is not a function (it is undefined)`

Reply:
@andrewcoelho Thanks for the report.

Strange... So react native has a `window` object but it is not an `EventTarget`?

```js
getBrowserGlobal('window')?.addEventListener?.('pagehide', this.#onPageHide);
```

Can you verify if that works on RN?

Reply:
@arv yes that fixes it (had to add to the `getBrowserGlobal('window')?.removeEventListener` below as well. For context this is react-native with expo, not sure if expo is doing something with window since it's also designed to run on the web.

Reply:
I'll add a PR that fixes this. (unless you want to do it?)

Reply:
sure will do

Reply:
@arv: https://github.com/rocicorp/mono/pull/4037